tests/test_github_api.py

```diff
@@ -0,0 +1,21 @@
+from prioritizator import Task
+
+
+def test_set_and_get_title():
+    task = Task(id=1)
+    expected_title = "Hola Mundo"
+    task._set_title(expected_title)
+    obtained_title = task._get_title()
```

Comment:
Aquí es donde podemos utilizar mockear la función `requests.get` y pedir que regrese un diccionario con una llave "title". Eso nos aseguraría que nos regrese sol que esperamos que haga el método `_get_title`. Yo me imagino que un cambio así podría servir:
```suggestion
def requests_get(self):
        return (
            {"title": "desde el imitador"}
        )

    mocker.patch(
        "requests.get", requests_get
    )
    obtained_title = task._get_title()
```

Reply:
"Recuerda" que deberás de instalar `pytest-mock` y agregar `mocker` como variable de entrada de la prueba `def test_set_and_get_title(mocker):`

Reply:
[Aquí](https://github.com/nepito/clean-enoa/pull/1/files#diff-a9977c909a1236b5452f8ba03a521f6bbd629a84ccec71c9659b120cb58e0c2fR40) es como yo le hice.